Spring-basics-tutorial/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+server.port = 8081
+spring.jpa.hibernate.ddl-auto=update
+management.endpoint.health.show-details=always
+management.endpoints.web.exposure.include=*
+management.endpoint.beans.enabled=true
+management.server.port=9001
```

Comment:
Please, consider adding a line break at the end of the files (EOF). It's useful when you print file's content using Linux `cat` utility. 
Let's take a look at example. Image you'd like to print the file content at do `ls -a` after.
Without line break at the end of file, it'd look like this:
```
[root@host # ] cat application.properties
server.port = 8081
spring.jpa.hibernate.ddl-auto=update
management.endpoint.health.show-details=always
management.endpoints.web.exposure.include=*
management.endpoint.beans.enabled=true
management.server.port=9001 [root@host # ] ls -a
```
 


While with line break at the end, it'd look like this:
```
[root@host # ] cat application.properties
server.port = 8081
spring.jpa.hibernate.ddl-auto=update
management.endpoint.health.show-details=always
management.endpoints.web.exposure.include=*
management.endpoint.beans.enabled=true
management.server.port=9001
[root@host # ]
```
In case the last string is really long, it'd become really uncomfortable. For example:
```
[root@host # ] cat application.properties
...management.server.port=9001management.server.port=9001management.server.port=9001management.server.port=9001management.server.port=9001management.server.port=9001management.server.port=9001 [root@host # ]
```